docs/source/API/core/view/resize.md

```diff
@@ -104,6 +145,20 @@ void resize(const I& arg_prop, Kokkos::View<T, P...>& v,
   * `layout`: a layout instance containing the new dimensions.
   * `arg_prop`: View constructor property, e.g., `Kokkos::WithoutInitializing`.
 
+* ```c++
+  template <class T, class... P, class... ViewCtorArgs>
+  void resize(const Impl::ViewCtorProp<ViewCtorArgs...>& arg_prop,
+         Kokkos::View<T, P...>& v,
+         const typename Kokkos::View<T, P...>::array_layout& layout);
+  ```
+  Resizes `v` to have the new dimensions while preserving the contents for the common subview of the old and new view. The new `Kokkos::View` is constructed using the View constructor properties `arg_prop`, e.g., `Kokkos::view_alloc(Kokkos::WithoutInitializing)`.
```

Comment:
What about use of execution space if provided to copy elements

Reply:
I added a sentence to both overloads. What I really want to say, though, is that providing an execution space follows standard semantics, i.e., use it as much as possible internally and enqueue operations without explicitly fencing. I'm not quite sure if that needs extra documentation and if so where a good place at a central location for such a statement would be.